ui/cypress/integration/admin.test.ts

```diff
@@ -7,11 +7,85 @@ describe('Use Admin tab', () => {
     cy.createInfluxDBConnection()
     cy.get('@connections').then(sources => {
       sourceId = sources[0].id
-      cy.request('GET', `/chronograf/v1/sources/${sourceId}/users`).then(
-        ({body: responseBody}) => {
-          console.log(responseBody.users)
-        }
-      )
+      url = `/sources/${sourceId}`
+    })
+  })
+
+  describe('Chronograf', () => {
```

Comment:
Consider splitting the file to `admin_influxdb.test.ts` and `admin_chronograf.test.ts`, unrelated functionalities are tested.